torch/ao/quantization/_pt2e/quantizer/qnnpack_quantizer.py

```diff
@@ -255,13 +258,15 @@ def annotate(self, model: torch.fx.GraphModule) -> torch.fx.GraphModule:
             # supported op type.
             self._annotate_conv2d_relu(node, global_config)
             self._annotate_conv2d(node, global_config)
-            self._annotate_linear(node, global_config)
+            self._annotate_linear3d(node, global_config)
+            self._annotate_linear2d(node, global_config)
             self._annotate_maxpool2d(node, global_config)
             self._annotate_add_relu(node, global_config)
             self._annotate_add(node, global_config)
             self._annotate_hardtanh(node, global_config)
             self._annotate_mean(node, global_config)
             self._annotate_adaptive_avg_pool2d(node, global_config)
+            self._annotate_flatten(node, global_config)
```

Comment:
I thought we were gonna propagate this via separate util?

Reply:
that's in a separate PR